internal/alb/tg/targetgroup.go

```diff
@@ -128,14 +133,15 @@ func (controller *defaultController) newTGInstance(ctx context.Context, name str
 	return tgInstance, nil
 }
 
-func (controller *defaultController) reconcileTGInstance(ctx context.Context, instance *elbv2.TargetGroup, serviceAnnos *annotations.Service) (*elbv2.TargetGroup, error) {
+func (controller *defaultController) reconcileTGInstance(ctx context.Context, instance *elbv2.TargetGroup, serviceAnnos *annotations.Service, healthCheckPort *string) (*elbv2.TargetGroup, error) {
```

Comment:
better to use string as argument, and pass to underline api with aws.String(healthCheckPort)